atlassian/atlassian-workflow.xml

```diff
@@ -0,0 +1,114 @@
+<Workflow name="Qmasters Atlassian workflow for QRadar" version="1.0" xmlns="http://qradar.ibm.com/UniversalCloudRESTAPI/Workflow/V2">
+    <Parameters>
+        <Parameter name="orgID" label="Organization ID" required="true" />
+        <Parameter name="apiToken" label="API Token" required="true" secret="true" />
+        <Parameter name="query" label="Query" required="false" default="" />
+        <Parameter name="timeZone" label="Time Zone" required="false" default="UTC" />
+        <Parameter name="startTime" label="Start Time" required="false" default="" />
+    </Parameters>
+
+    <Actions>
+        <!-- Clear the log source status before a new workflow run starts -->
+        <ClearStatus />
+
+        <!-- Initialize the risingValue filter for the first run. -->
+        <If condition="count(/startTime) > 0" >
+            <Initialize path="/bookmark" value="${/startTime}"/>
+        </If>
+        <Else>
+            <Initialize path="/bookmark" value="${time() - 3600000}" />
+        </Else>
+
+        <!-- Set the firse page number -->
+        <Set path="/pageNumber" value="1"/>
+
+        <!-- Get events -->
+        <CallEndpoint url="https://api.atlassian.com/admin/v1/orgs/${/orgID}/events" method="GET" savePath="/getEvents">
+            <BearerAuthentication token="${/apiToken}" />
+            <QueryParameter name="from" value="${/bookmark}" omitIfEmpty="true"/>
+            <QueryParameter name="cursor" value="${/cursor}" omitIfEmpty="true"/>
+            <QueryParameter name="query" value="${/query}" omitIfEmpty="true"/>
+            <QueryParameter name="order" value="desc"/>
+        </CallEndpoint>
+
+        <!-- Handle Errors -->
+        <If condition="/getEvents/status_code != 200">
+            <Log type="ERROR" message="Atlassian API error: ${/getEvents/body}"/>
+            <Abort reason="Atlassian abort reason: ${/getEvents/body}" />
+        </If>
+
+        <!-- Post Events, if any -->
+        <If condition="count(/getEvents/body/data) > 0">
+            <PostEvents path="/getEvents/body/data" source="${/orgID}" />
+        </If>
+
+        <!-- Prepare the lastEventsTime path -->
+        <Set path="/lastEventsTime" value="[]"/>
+
+        <!-- Merge the log data from the API response into the "/lastEventsTime" path. -->
+        <Merge targetPath="/lastEventsTime" sourcePath="/getEvents/body/data/attributes/time" />
+
+        <!-- Set the number of events -->
+        <Set path="/currentFetchEvents" value="${count(/getEvents/body/data/id)}"/>
+        <Set path="/numberOfEvents" value="${count(/getEvents/body/data/id)}"/>
```

Comment:
```suggestion
        <!-- Set the number of events -->
        <Set path="/currentFetchEvents" value="${count(/getEvents/body/data/id)}"/>
        <Set path="/numberOfEvents" value="${/currentFetchEvents}"/>

        <!-- Post Events, if any -->
        <If condition="/currentFetchEvents > 0">
            <PostEvents path="/getEvents/body/data" source="${/orgID}" />
        </If>

        <!-- Prepare the lastEventsTime path -->
        <Set path="/lastEventsTime" value="[]"/>

        <!-- Merge the log data from the API response into the "/lastEventsTime" path. -->
        <Merge targetPath="/lastEventsTime" sourcePath="/getEvents/body/data/attributes/time" />
```
You are using here the `count()` function 3 times:
1. For checking whether any data has returned.
2. To get the number of fetched events.
3. To initialize the total number of fetched events.

Reply:
fixed